aten/src/ATen/gen.py

```diff
@@ -291,7 +293,7 @@ def generate_storage_type_and_tensor(backend, density, declarations, per_op_regi
     top_env['type_ids'].append(tag + ',')
 
     env['legacy_th_headers'] = []
-    if backend == 'CUDA':
+    if backend_to_devicetype(backend) == 'CUDA':
```

Comment:
It's probably clearer if you just have a `is_cuda_backend` test

Reply:
Agree, fixed.